gitgud/operations.py

```diff
@@ -108,6 +108,21 @@ def create_tree(self, commits, head):
         if head_is_commit:
             self.repo.git.checkout(commit_objects[head])
 
+    # Revert detection helper function
+    @staticmethod
+    def find_Revert(commit_name):
```

Comment:
change this method definition to `parse_name(commit_msg)` since it's not just looking for the revert stuff, it's also handles other cases - aka parsing. It's also taking in the message, not the "name". 

Also, as a note, the first line of a commit message is what's considered to be the "title" and is what you see when you type `git log --oneline`

Reply:
Okay, changed and pushed!